content/en/blog/_posts/XXXX-XX-XX-dra-133-update.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+title: "Kubernetes 1.33: New features in DRA"
+slug: dra-133-updates
+draft: true
+date: XXXX-XX-XX
+author: >
+  [Morten Torkildsen](https://github.com/mortent) (Google)
+  [Patrick Ohly](https://github.com/pohly) (Intel)
+---
+
+Kubernetes [Dynamic Resource Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA) was originally introduced as an Alpha feature in the 1.26 release, and then went through a significant redesign for Kubernetes 1.31. The main DRA feature went to beta in 1.32 and the project hopes it will be generally available in Kubernetes 1.34.
```

Comment:
nit:
```suggestion
Kubernetes [Dynamic Resource Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA) was originally introduced as an alpha feature in the v1.26 release, and then went through a significant redesign for Kubernetes v1.31. The main DRA feature went to beta in v1.32 and the project hopes it will be generally available in Kubernetes v1.34.
```